content/influxdb/clustered/write-data/best-practices/data-lifecycle.md

```diff
@@ -52,32 +78,34 @@ metadata:
   namespace: influxdb
 spec:
   package:
-    # Surrounding configuration parameters have been snipped for brevity
+    # ...
     spec:
       components:
        garbage-collector:
          template:
            containers:
              iox:
                env:
-                 INFLUXDB_IOX_GC_OBJECTSTORE_CUTOFF: '6h',
-                 INFLUXDB_IOX_GC_PARQUETFILE_CUTOFF: '6h',
+                 INFLUXDB_IOX_GC_OBJECTSTORE_CUTOFF: '6h'
+                 INFLUXDB_IOX_GC_PARQUETFILE_CUTOFF: '6h'
```

Comment:
I wasn't sure about this environment variable format here. I'm fairly certain the commas are invalid yaml, but also, all the k8s documentation I can find uses the following format for environment variables:

```yml
env:
  - name: INFLUXDB_IOX_GC_OBJECTSTORE_CUTOFF
    value: '6h'
  - name: INFLUXDB_IOX_GC_PARQUETFILE_CUTOFF
    value: '6h'
```

Reply:
Ah yeah! That's my bad :facepalm: 

So the commas are definitely incorrect, that comes from me copying these variables for a test we have internally, _but_ that is in Jsonnet so it includes commas which I forgot to strip out.

The mapping as-is here for `env` is correct though. It's part of how the Clustered operator interprets the given manifest - it follows the same pattern as our upcoming helm chart too so that they're kept aligned. We do this in a similar way to `atlantis` ([like this](https://github.com/runatlantis/helm-charts/blob/3116f468962206d14c57c84bc9a37e51d1bb3b0f/charts/atlantis/values.yaml#L475-L477)) and other products where you can provide the key-value mapping without the extra boilerplate.